byway/src/main/webapp/destinations.css

```diff
@@ -4,9 +4,7 @@
   width: 650px;
 }
 
-
-#destinations-page #map img
-{
+#destinations-page #map img{
   max-width : none;
 }
 
```

Comment:
Small nit: try to be consistent with spacing throughout this file when setting the attributes. ie adding a space after the colon or having no space.

Reply:
fixed